example.init.yaml

```diff
@@ -101,14 +114,29 @@ root_domain: "myfaas.club"
 ## Keep active if using a cluster with a LoadBalancer available.
 ingress: loadbalancer
 
-
 ## Populate from GitHub App
 github:
   app_id: "24304"
 
+## GitLab
+gitlab:
+  gitlab_instance: "https://gitlab.o6s.io/"
+  ## The OpenFaaS Cloud instance will build repositories with the given tag.
+  #  Can be custom populated with required value.
+  installation_tag: "openfaas-cloud"
```

Comment:
I don't think this is actually doing anything, should be referenced in plan type and be generated with template

Reply:
Hm, I think I removed it. Seems like it's forgotten here

Reply:
👌